ci/scripts/compare_folders.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
```

Comment:
Where are we using this?

Reply:
I understand. 
You can maintain this in a branch on your fork.
We will need to discuss how to do comparisons (and what to compare) in a different issue.

Reply:
Agreed. PRs need to be one thing at a time as much as possible. Adding extra stuff you happened to do in the meantime just makes reviews more difficult, which means they take longer, and the increase is not linear (and I'm just as guilty of this as anyone else).